modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -122,7 +123,11 @@ public void populateConfigJSONObject(
 			jsonObject.put("resize_dir", "vertical");
 		}
 
-		jsonObject.put("resize_enabled", resizable);
+		jsonObject.put(
+			"enableBalloonEditor", FFBalloonEditorConfigurationUtil.enable()
```

Comment:
I think this should be in `CKEditorBalloonEditorConfigContributor`. Other versions of CKEditor don't care about the FF.

Sidnote: I dislike that a boolean is named as action (`enable*`) instead of an adjective (`*Enabled`), but this follows  backend naming so I guess it's fine :disappointed:

Reply:
> Other versions of CKEditor don't care about the FF

Yes, but I removed the 

```java
	@Override
	public void populateConfigJSONObject(
```

while migrating things on BalloonEditor to use `DefaultBalloonEditorConfiguration.js`. Let's see how it goes

Reply:
It worked!